frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,22 @@
+import { Navigate } from 'react-router-dom';
+
+import { AppRoute } from '~/libs/enums/enums.js';
+
+type Properties = {
+  component: React.FC;
+};
+
+const ProtectedRoute: React.FC<Properties> = ({
```

Comment:
Please use <Outlet/> and create the example route called '/protected'

Reply:
Of course, thank you.